streaming/streaming.go

```diff
@@ -71,15 +73,20 @@ type DatastreamCfg struct {
 }
 
 type DataflowCfg struct {
-	JobName             string            `json:"jobName"`
-	Location            string            `json:"location"`
-	HostProjectId       string            `json:"hostProjectId"`
-	Network             string            `json:"network"`
-	Subnetwork          string            `json:"subnetwork"`
-	MaxWorkers          string            `json:"maxWorkers"`
-	NumWorkers          string            `json:"numWorkers"`
-	ServiceAccountEmail string            `json:"serviceAccountEmail"`
-	DbNameToShardIdMap  map[string]string `json:"dbNameToShardIdMap"`
+	ProjectId            string            `json:"projectId"`
+	JobName              string            `json:"jobName"`
+	Location             string            `json:"location"`
+	VpcHostProjectId     string            `json:"vpcHostProjectId"`
+	Network              string            `json:"network"`
+	Subnetwork           string            `json:"subnetwork"`
+	MaxWorkers           string            `json:"maxWorkers"`
+	NumWorkers           string            `json:"numWorkers"`
+	ServiceAccountEmail  string            `json:"serviceAccountEmail"`
+	MachineType          string            `json:"machineType"`
+	AdditionalUserLabels string            `json:"additionalUserLabels"`
+	KmsKeyName           string            `json:"kmsKeyName"`
+	GcsTemplatePath      string            `json:"gcsTemplatePath"`
+	DbNameToShardIdMap   map[string]string `json:"dbNameToShardIdMap"`
 }
```

Comment:
Just so that I remember the usage here - what is the difference between `dataflowCfg` and `DataflowConfig`?

Reply:
They are largely similar with DataflowConfig being a subset of dataflowCfg. 
- DataflowConfig is used in web.go primarily to unmarshall request bodies and set session state. 
- DataflowCfg is whats used in the backend for orchestrating streaming migrations.

They can be deduped but is a slightly more involved changed so have'nt taken that as part of this.

Reply:
created b/312106652